src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -104,6 +105,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ov::Model>
     ov::pass::Manager manager(get_pass_config());
     manager.set_per_pass_validation(false);
     using namespace ov::pass;
+    REGISTER_PASS(manager, RemoveRtInfo)
     REGISTER_PASS(manager, InitNodeInfo)
```

Comment:
maybe we need to merge with `InitNodeInfo` to reduce number of passes across all the model?

Reply:
tried to do so, it causes problems in unit tests. TransformationTestsF::SetUp() uses InitNodeInfo() transformation. Some of the unit tests tries to set rt_info attributes. If InitNodeInfo cleans everything they fail

Reply:
As suggestion, I could try to remove InitNodeInfo from TransformationTestsF::SetUp(). Need to understand why it is needed there

Reply:
@itikhono found that there are attributes, that could be loaded from IR. They should be anyway passed without removing. So we need to check if attribute should be removed or not.

Reply:
There are still a lot of errors in unit tests. Investigating

Reply:
If we want to remove InitNodeInfo, we also should remove check_rt_info from all unit tests